apptrail/README.md

```diff
@@ -0,0 +1,65 @@
+# Apptrail
+
+## Overview
+
+[Apptrail][1] is the comprehensive SaaS audit logs platform. SaaS companies use Apptrail to add full-featured customer-facing audit logs to their products. Customers can view, query, analyze, and export audit logs from all of their SaaS vendors using Apptrail.
+
+Apptrail's audit event streaming feature ([_trails_][11]) lets you deliver audit logs to dozens of popular destinations.
+
+The Apptrail Datadog integration allows you to continously export your SaaS audit logs from Apptrail to Datadog in realtime. You can use Datadog to analyze, archive, monitor, and alert on your SaaS audit logs.
+
+## Setup
+
+As a prerequisite, you should be signed up by your SaaS vendor for Apptrail.
+
+To get started, create a delivery trail in the Apptrail Portal and choose Datadog as the configured destination.
+
+For a complete guide and set of steps to follow, consult the [Apptrail Datadog delivery documentation][2].
```

Comment:
I would move this to the end of the Setup section since you list out the steps below as well. 
```suggestion
For more details, take a look at the [Apptrail Datadog delivery documentation][2].
```